src/components/accounts/RecoverWithLink.js

```diff
@@ -92,17 +111,32 @@ class RecoverWithLink extends Component {
         this.state = {
             accountId: this.props.accountId,
             seedPhrase: this.props.seedPhrase,
-            successSnackbar: false
+            successSnackbar: false,
+            successView: true
         };
     }
 
+    componentWillMount = () => {
+        if (!this.isLegit) {
+            this.setState({ successView: false });
```

Comment:
if you want to create derived property, why not just create it like this?

```
get successView() {
      return this.isLegit;
}
```

trying to track it in state is far more error prone

Reply:
Very nice